.github/merge_rules.json

```
@@ -81,6 +81,7 @@
    },
    {
       "name": "OSS CI / pytorchbot / XLA",
+      "COMMENT": "here is dedicated section for XLA since time to time we need to disable XLA only.",
```

Minor nits: Consider lower casing to `"comment"` for visual consistency.

In fact, we could even call this the `"description"` section (since it seems like that's what you want to use it as for now).  It'll still be ignored by our parsers (since they don't expect it) but it reads better and we could theoretically integrate it with some other tooling in the future.

```suggestion
      "description": "here is dedicated section for XLA since time to time we need to disable XLA only.",
```